HKMP/Animation/AnimationEffect.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Hkmp.Util;
```

Comment:
Not sure what every change is on the first line of the files, but my IDE started complaining about file encodings for a bit.